src/hdinsightonaks/HISTORY.rst

```diff
@@ -3,6 +3,19 @@
 Release History
 ===============
 
+1.0.0b2
+++++++
+* Cluster Pool
+Enable create cluster pool with user network profile.
+Enable get cluster pool available upgrade versions.
+* Cluster
+Enable create Ranger cluster.
+Enable get cluster available upgrade versions.
+Enable set internal ingress.
+Enable check if the cluster name is available.
+Enable upgrade hot fix for cluster.
+Enable upgrade node os for cluster.
+
```

Comment:
It is better to specify the corresponding affected commands and list all of them separately using `*`.
You can refer to the [HISTORY.rst](https://github.com/Azure/azure-cli-extensions/blob/c4baba3eca9b40aec34067991651a8a59cddc289/src/containerapp/HISTORY.rst?plain=1).

Reply:
Updated in the recommended format.